docs/products/postgresql/concepts/high-availability.md

```diff
@@ -51,26 +55,27 @@ primary node.
 
 ## Highly available Business and Premium service plans
 
-When the failed node is a PostgreSQL **standby** node, the primary node
+When the failed node is a PostgreSQL standby node, the primary node
 keeps running normally and provides a normal service level to the client
 applications. Once the new replacement standby node is ready and
 synchronised with the primary node, it starts replicating the primary
 node in real time as the situation reverts back to normal.
 
-When the failed node is a PostgreSQL **primary** node, the combined
+When the failed node is a PostgreSQL primary node, the combined
 information from the Aiven monitoring infrastructure and the standby
 node is used to make a failover decision. The standby node is then
 promoted as the new primary and immediately starts serving clients. A
 new replacement node is automatically scheduled and becomes the new
-standby node.
+standby node. There is no data loss in this scenario.
```

Comment:
I think there is always a risk for loss of data. There might be edge cases where the standby is behind the master or the promotion of the standby might take longer. I think we should soften this or find another perspective to create contrast between this scenario and the previous one (standby fail).

Reply:
I agreem I also made a similar comment.